### PR TITLE
OE-567: UX Review

### PR DESCRIPTION
**Pull request by kristojorg**

### CODE REVIEW

File: src/marketing/components/Carousel/index.tsx [hunk 1]
```diff
@@ -36,30 +36,29 @@ const CarouselSection: React.FC<{
...
         <GridItem>
-          <Text textStyle="h1" as="h1" mt={[3, 4, 6]}>
+          <Text textStyle="h1" as="h1" mt={{ base: 3, sm: 4, md: 6 }}>
             {title}
           </Text>
         </GridItem>
-        <GridItem gridColumn={["1/3", null, "2 / 3"]} gridRow="2 / 3">
-          <Text fontSize={["md", "xl"]} maxW="34em">
+        <GridItem gridColumn={{ base: "1/3", md: "2 / 3" }} gridRow="2 / 3">
+          <Text fontSize={{ base: "md", md: "xl" }} maxW="34em">
```
Comment (samanthaandrews) [inline src/marketing/components/Carousel/index.tsx:61]: Wondering if we can change this to `sizes.lg` from the theme file? It's 34rem though.

File: src/marketing/components/Header.tsx [hunk 2]
```diff
@@ -36,11 +44,11 @@ const Header: React.FC = () => {
       top={0}
       left={0}
       as="header"
+      height="header"
```
Comment (samanthaandrews) [inline src/marketing/components/Header.tsx:47]: nice!

**Review by samanthaandrews — APPROVED** [review]
I'm curious about the change from `menu` to `drawer` in the menu. This looks good to me though overall!

**Review by Toxiapo — APPROVED** [review]
Looks great! I am also curious about the switch from menu component to the drawer, [accessibility wise](https://accessibility.oit.ncsu.edu/it-accessibility-at-nc-state/developers/accessibility-handbook/mouse-and-keyboard-events/menus/), either one works, it just the matter of navigate with arrow keys or tabs for the end-users.

I also did some UX updates with Jen's feedback, if the codes are ok, this is ready to get merged.  @kristojorg

**Pull request merged**